### PR TITLE
LRAC-5773 Create read plugin

### PR DESCRIPTION
**Pull request by fcodias**

### CODE REVIEW

File: modules/apps/analytics/analytics-client-js/src/utils/constants.js [hunk 1]
```diff
@@ -37,3 +37,19 @@ export const STORAGE_KEY_USER_ID = 'ac_client_user_id';
 export const LIMIT_FAILED_ATTEMPTS = 7;
 
 export const REQUEST_TIMEOUT = 5000;
+
+// Language Constants
+
+export const LANGUAGE_JAPANESE = 'ja';
```
Comment (andreldm) [inline modules/apps/analytics/analytics-client-js/src/utils/constants.js]: Maybe `LOGOGRAPHIC_LANGUAGES = new Set(['ja', 'ko', 'zh'])` ?

File: modules/apps/analytics/analytics-client-js/src/plugins/read.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+		readTracker.setExpectedViewDuration(
+			() => analytics.send('pageRead', applicationId),
+			getExpectedViewDuration(content)
+		);
+	});
+
+	const onScroll = debounce(() => {
+		scrollTracker.onDepthReached((depth) => {
+			if (depth >= 75) {
```
Comment (andreldm) [inline modules/apps/analytics/analytics-client-js/src/plugins/read.js]: magic number

**Comment by andreldm** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/analytics/analytics-client-js/src/utils/constants.js [hunk 3]
```diff
@@ -48,18 +48,14 @@ export const MARK_PAGE_LOAD_TIME = 'pageLoadTimeSPA';
...
-
 // Read metrics Constants
 
+export const LOGOGRAPHIC_LANGUAGES = new Set(['ja', 'ko', 'zh']);
+
 export const CHARS_PER_MIN = 500;
 
+export const MIMIMUN_SCROLL_DEPTH = 75;
+
 export const WORDS_PER_MIN = 265;
```
Comment (andreldm) [inline modules/apps/analytics/analytics-client-js/src/utils/constants.js]: Sort and maybe use a common prefix?

Comment (fcodias) [reply modules/apps/analytics/analytics-client-js/src/utils/constants.js]: updated @andreldm!

**Comment by fcodias** [comment]
ci:test:sf

**Comment by andreldm** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by andreldm** [comment]
Pull request submitted to https://github.com/andreldm/liferay-portal/pull/26. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**